### ISSUE TITLE
[Consistency] User encounters content with errors or inconsistencies.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#77116]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Accredited Representation Management Team
**Product:** Find a Representative (MVP)
**Feature:** Find a Representative

## Findings details
**VA.gov Experience Standard - issue:** User encounters content with errors or inconsistencies.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When selecting a "Sort by" option, the "Showing X - Y of Z..." message updates to match --- before the filter is applied by pressing the "Apply" button. This makes the "Showing" text a false statement and is potentially confusing. It may especially impact low vision users at high screen magnification, who may see the "Showing" text and the "Sort by" filter but don't see the results until the scroll down to them.

## Recommended action
Recommend that the "Showing" text doesn't update until after the filters are applied.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Browser zoom
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
